daemon/info.go

```diff
@@ -26,12 +26,16 @@ import (
 // SystemInfo returns information about the host server the daemon is running on.
 func (daemon *Daemon) SystemInfo() (*types.Info, error) {
 	kernelVersion := "<unknown>"
-	if kv, err := kernel.GetKernelVersion(); err == nil {
+	if kv, err := kernel.GetKernelVersion(); err != nil {
+		logrus.Errorf("Could not get kernel version: %v", err)
```

Comment:
Should this be an error, or a warning? If this is just informational, I wonder if we really need this (`<unknown>` already indicates that we were unable to determine, so I wonder if we need the additional noise)